.readthedocs.yml

```diff
@@ -1,23 +1,24 @@
-# .readthedocs.yml
-# Read the Docs configuration file
-# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
-
-# Required
 version: 2
 
-# Build documentation in the docs/ directory with Sphinx
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.11"
```

Comment:
updating to 3.11 is cool!

ci should probably do the same to avoid surprises, afaict:

- gha: https://github.com/graphistry/pygraphistry/blob/c93bd5d4b8a1f85bc00fbd8056d62f1a459bd50d/.github/workflows/ci.yml#L287C23-L287C23
- the (old) sphinx container it runs: https://github.com/graphistry/pygraphistry/blob/c93bd5d4b8a1f85bc00fbd8056d62f1a459bd50d/docs/docker.sh#L15
- which is py3.8: https://github.com/graphistry/pygraphistry/actions/runs/5950967983/job/16319302664?pr=496#step:3:100